package.json

```diff
@@ -40,7 +40,7 @@
     "@semantic-release/npm": "^7.1.3",
     "@semantic-release/release-notes-generator": "^9.0.1",
     "conventional-changelog-conventionalcommits": "^4.6.0",
-    "eslint": "^6.8.0",
+    "eslint": "^7.32.0",
```

Comment:
Not the most up-to-date version. I just wanted support for optional chaining so it isn't upset when I do `?.` which is cleaner than nested conditionals.